USER-GUIDE.md

```diff
@@ -0,0 +1,484 @@
+# KCP-Edge User Guide:
+
+## Required Packages:
+   - docker
+   - kind
+   - kubectl (version range expected: 1.23-1.25)
+
+
+## Setup Instructions
+
+This guide is intended to explain how KCP-Edge works with a detailed example.
+
+### 1. Create your own edge infrastructure (edge clusters) - Bring Your Own Clusters (BYOC)
+
+Create your edge clusters or bring your own k8s edge clusters. In this example, we deploy only a single cluster using [kind](https://kind.sigs.k8s.io/). Our edge cluster is named “florin”:
```

Comment:
The "or" in the first sentence suggests that there are two alternatives being suggested, but as far as I can tell the two sides of the "or" mean the same thing.  If there really is only one idea here, then the first clause is a great way to describe it and no more is needed.

Reply:
Fixed